src/App.tsx

```diff
@@ -1,25 +1,51 @@
 import React from 'react';
-import { useSelector } from 'react-redux';
+import { useSelector, useDispatch } from 'react-redux';
 
 import './App.scss';
-import Start from './components/Start';
-import { Finish } from './components/Finish';
+import { TodoList } from "./components/TodoList";
+import { getPreparedData } from "./helpers/api";
+import { isLoaded, isLoading, setTodos, getloading, getloaded, getTodos } from "./store";
 
-import { isLoading, getMessage } from './store';
+// import { isLoading, getMessage } from './store';
```

Comment:
remove commented code, where is your attention?)

Reply:
Useful in the future, commented out, so as not to interfere)

Reply:
in what future?) keep it simple, if you use it uncomment, if not delete it